tee-worker/omni-executor/aa-contracts-client/src/omni_account_client.rs

```diff
@@ -75,6 +78,74 @@ impl<P: RpcProvider<Transaction = TransactionRequest>> OmniAccountClient<P> {
 			FixedBytes::<32>::abi_decode(&result).map_err(|_| error!("Could not decode owner"))?;
 		Ok(owner)
 	}
+
+	pub async fn add_passkey_signer(&self, pk: PasskeyPublicKey) -> Result<(), ()> {
+		let call_data = addPasskeySignerCall { pk }.abi_encode();
+		let tx = build_call_transaction(self.address, call_data);
+		self.rpc_client.send_transaction(tx).await.map_err(|_| ())?;
```

Comment:
These `self.rpc_client.send_transaction` calls probably won't work - as mentioned it won't pass the `onlyOwner()` check